sympy/core/compatibility.py

```diff
@@ -361,10 +360,8 @@ def is_sequence(i, include=None):
             bool(include) and
             isinstance(i, include))
 
-try:
-    from itertools import zip_longest
-except ImportError:  # Python 2.7
-    from itertools import izip_longest as zip_longest
+
+from itertools import zip_longest
```

Comment:
This should be removed, and the imports that use it changed to import it directly from itertools (same with all other Python 2 things in this file).

Reply:
Can you please rephrase the sentence?  What I can understand from your statement is that the file using `zip_longest` for instance, should import that directly from `itertools` but not from `sympy.core.compatibility`.